iiif_prezi3/helpers/annotation_helpers.py

```diff
@@ -0,0 +1,40 @@
+from ..loader import monkeypatch_schema
+from ..skeleton import (Annotation, AnnotationPage, Canvas, Collection,
+                        Manifest, Range)
+
+
+class AnnotationHelpers:
+
+    def add_annotation(self, annotation):
+        """Adds the annotation object to the (AnnotationPage object in the) annotations property.
+
+        Args:
+          annotation (iiif-prezi3.skeleton.Annotation): the Annotation to add
```

Comment:
This should be `iiif_prezi3` as that's the module's name (vs the project which is with the -)

Also it might be better to skip the skeleton part if it's possible, as we need users to import using the chained import triggered by the main module's `__init__.py` so abstracting away the detail will lessen the chance of an accidental error if users follow the autogenerated docs.

tl;dr: I think this should be `annotation (iiif_prezi3.Annotation)` but with the caveat that I don't know how this will work with the auto doc generator.

Reply:
Good point. (I blindly copy-pasted from canvas_helpers.py ;-)